Cargo.toml

```diff
@@ -179,11 +180,11 @@ default-members = ["cmd"]
 [profile.dev]
 opt-level = 0
 debug = 1 # required for line numbers in tests, see tikv #5049
-codegen-units = 4
+codegen-units = 16
```

Comment:
cc @brson did you measure the perf trade-off here?

Reply:
I did not measure the perf trade-off. IMO, it is fine even if the perf is not good since dev build only be used in tests, not in prod.